docs/customization/plugin-api.md

```diff
@@ -223,7 +223,7 @@ const thisVariableWillBeNull = getComponent("not_real", false, { failSilently: t
 
 #### Wrap-Actions
 
-Wrap Actions allow you to override the behavior of an action in the system.
+Wrap Actions allow you to override the behavior of the action in the system.
```

Comment:
`an` is more correct, as there are many possible actions.

Reply:
ok